app/Providers/AppServiceProvider.php

```diff
@@ -10,7 +10,8 @@
 class AppServiceProvider extends ServiceProvider
 {
     protected $repositories = [
-        'WebhookRepository'
+        'WebhookRepository',
+        'BotRepository'
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoCommaAfterLast</br>**Severity**: error</br>**File**: app/Providers/AppServiceProvider.php</br>**Line**: 14</br>Comma required after last value in array declaration